neurolib/models/multimodel/builder/base/backend.py

```diff
@@ -83,26 +83,26 @@ class NumbaBackend(BaseBackend):
     DEFAULT_DT = 0.1  # in ms
 
     CURRENT_Y_REGEX = r"current_y\([0-9]*\)"
-    CURRENT_Y_NUMBA = "y[max_delay + i - 1, {idx}]"
+    CURRENT_Y_NUMBA = "y[{idx}, max_delay + i - 1]"
```

Comment:
to avoid confusion I switched multimodel backends to `(space x time)` axes order